src/components/modules/shortcuts.ts

```diff
@@ -62,4 +62,11 @@ export default class Shortcuts extends Module {
     this.registeredShortcuts[index].remove();
     this.registeredShortcuts.splice(index, 1);
   }
+
+  /**
+   * Remove all shortcuts
+   */
+  public removeAll() {
+    this.registeredShortcuts = [];
```

Comment:
to remove shortcut, you can call `shortcut.remove()`

see https://github.com/codex-team/codex.shortcuts

Reply:
Same line of reasoning as Listeners. Should any shortcuts remain active in read-only mode?

Reply:
yes, for example cmd+c, cmd+a